controller/sentry/service.go

```diff
@@ -0,0 +1,41 @@
+package sentry
+
+import (
+	"context"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/giantswarm/microerror"
+)
+
+type Config struct {
+	Dsn string
+}
+
+type Sentry struct {
+	enabled bool
+}
+
+func New(config Config) (*Sentry, error) {
+	disabled := Sentry{enabled: false}
```

Comment:
I know I know but more elegant solution would be to have an interface and two implementations. One doing nothing and another doing something.

Reply:
Make sense. Done.